python/get_config.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python -B
+
+# Insert the parent directory into the module import search path.
+import os
+import sys
+sys.path.insert(1, os.path.dirname(sys.path[0]))
```

Comment:
Yah, it doesn't seem necessary, but something blew up when I tried to remove it awhile ago.  This is from before when I was involved.